optimize.c

```diff
@@ -2057,7 +2057,7 @@ convert_code_r(compiler_state_t *cstate, conv_state_t *conv_state,
 		}
 	}
 	src = p->stmts;
-	for (off = 0; off < slen && src; off++) {
+	for (off = 0; off < (u_int) slen && src; off++) {
```

Comment:
OK. I removed that modification at: https://github.com/the-tcpdump-group/libpcap/pull/512/commits/c2d601298b8f57a18fe21ae735360d92e5ebc487